packages/typescript-estree/tests/lib/persistentParse.test.ts

```diff
@@ -148,47 +176,69 @@ function baseTests(
     // write a new file and attempt to parse it
     writeFile(PROJECT_DIR, bazSlashBar);
 
-    expect(() => parseFile(bazSlashBar, PROJECT_DIR)).not.toThrow();
+    expect(() => {
+      parseFile(bazSlashBar, PROJECT_DIR);
+    }).not.toThrow();
   });
 
   it('allows renaming of files', () => {
     const PROJECT_DIR = setup(tsConfigIncludeAll, true);
     const bazSlashBar = 'baz/bar' as const;
 
     // parse once to: assert the config as correct, and to make sure the program is setup
-    expect(() => parseFile('foo', PROJECT_DIR)).not.toThrow();
+    expect(() => {
+      parseFile('foo', PROJECT_DIR);
+    }).not.toThrow();
     // bar should throw because it doesn't exist yet
-    expect(() => parseFile(bazSlashBar, PROJECT_DIR)).toThrow();
+    expect(() => {
+      parseFile(bazSlashBar, PROJECT_DIR);
+    }).toThrow();
```

Comment:
this is the sort of reason I think that the concise arrow function style is better - satisfying this lint rule just added some 160 lines to this file and (IMO) made the code *harder* to read